CHANGELOG.md

```diff
@@ -12,7 +12,9 @@
 - Add isar breadcrumbs ([#1800](https://github.com/getsentry/sentry-dart/pull/1800))
 - Starting with Flutter 3.16, Sentry adds the [`appFlavor`](https://api.flutter.dev/flutter/services/appFlavor-constant.html) to the `flutter_context` ([#1799](https://github.com/getsentry/sentry-dart/pull/1799))
 - Add beforeScreenshotCallback to SentryFlutterOptions ([#1805](https://github.com/getsentry/sentry-dart/pull/1805))
-
+- Add support for `readTransaction` in `sqflite` ([#1819](https://github.com/getsentry/sentry-dart/pull/1819))
+  - This method was introduced in `sqflite > 2.5.0+2`. For older versions, this plugin will call `transaction`
```

Comment:
@buenaflor @denrase 
No need TBH, readTransaction won't be called since the method does not exist before 2.5.0+2
the fallback is just to return something so the method compiles, it has to return something, and you could return null if possible.

Reply:
true, valid point then let's exclude the additional note

Reply:
Yeah, will probably just lead to confusion. Will remove, thank you